notificationworkerlambda/src/main/scala/com/gu/notifications/worker/utils/NotificationParser.scala

```diff
@@ -39,13 +39,7 @@ object NotificationParser extends Logging {
 
   private def parseChunkedTokens(json: JsValue): ChunkedTokens = {
     json.validate[ChunkedTokens] match {
-      case JsSuccess(chunkedTokens, _) => {
```

Comment:
The number of tokens each sqs message contains is now included in a downstream log event, so this isn't needed anymore